### DISCUSSION TITLE
Cannot load images in `public`

### DISCUSSION THREAD

**Discussion by cheetosysst** [vercel/next.js#45650] [Help]
I just upgraded my repo from next 12 to next 13 (^13.1.6), and the Images stopped working.

```jsx
<Image
	src={"/maraho_avatar.png"}
	alt="avatar"
	width={128}
	height={128}
/>
```

In the browser, the path is changed to 

```html
<img alt="avatar" srcset="/?url=%2Fmaraho_avatar.png&amp;w=128&amp;q=75 1x, /?url=%2Fmaraho_avatar.png&amp;w=256&amp;q=75 2x" src="/?url=%2Fmaraho_avatar.png&amp;w=256&amp;q=75" decoding="async" data-nimg="1" loading="lazy" style="" width="128" height="128">
```

The image won't load, it only shows the alt text.

In the network panel:

![image](https://user-images.githubusercontent.com/32976729/217209487-684cd560-c9cb-4a19-a309-41d81be005e1.png)

**Comment by cheetosysst** [comment]
Turns out I can just use a loader like

```jsx
export default function loader({ src, width, quality }) {
	return `${src}?w=${width}&q=${quality || 75}`;
}
```
or delete the `images` part of setting.

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
	reactStrictMode: true,
	images: {
		loader: 'akamai',
		path: "",
		domains: ["avatars.githubusercontent.com"],
	},
};

module.exports = nextConfig;
```

Not very familiar with `next/image`, so I'm really sure what is happening

**Comment by abedshaaban** [comment]
make sure you are importing `Image` from `import Image from 'next/legacy/image'`

that's how you should import it in nextjs 13.0
